Makefile

```diff
@@ -67,16 +70,28 @@ verify: fmt vet tidy generate ## verification checks against code.
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./util ./controllers -coverprofile cover.out
 
 .PHONY: test-e2e
-test-e2e: ## Run e2e tests.
-	@echo "TODO: PLEASE DEFINE test-e2e TARGET"
+test-e2e: ginkgo ## Run e2e tests.
```

Comment:
Is there a reason why `test-e2e` is not the main target for e2e testing?

Reply:
I looked at the PR as a whole, the `test-e2e` will basically run the tests locally. While `e2e` does the full thing. I still think `test-e2e` feels like the natural entrypoint.

Reply:
I can work on updating this so that everything runs as part of the `test-e2e` target

Reply:
Should be fixed with commit https://github.com/operator-framework/oria-operator/commit/9cfafd2158ea14da68bb62915c3defd4878b3ba2